pages/index.js

```diff
@@ -21,8 +21,11 @@ export default function Home() {
 					rel='icon'
 					href='/favicon.ico'
 				/>
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"></link>
```

Comment:
You shouldn't add font awesome icons here, you should use the font awesome react library.